src/model/modelinfo/TypeInfo.ts

```diff
@@ -0,0 +1,63 @@
+import Element from "./Element";
+import ElementFactory, { IElementXml } from "./ElementFactory";
+import { normalizeElementTypeName } from "../../utils";
+
+/**
+ * This is how the XML is structured for a TypeInfo element
+ * Example:
+ *
+ * <ns4:typeInfo
+ *     xsi:type="ns4:ClassInfo" namespace="FHIR" name="ActionSelectionBehavior" retrievable="false" baseType="FHIR.Element">
+ *   <ns4:element name="value" elementType="System.String"/>
+ * </ns4:typeInfo>
+ */
+export interface ITypeInfoXml {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5691315)